src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -59,12 +60,28 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     List<Message> messages = datastore.getMessages(user);
+    List<Message> messages1 = new ArrayList<>();
```

Comment:
Can you rename this variable to something more specific than "messages1" (for example, "messagesWithImages")?

Reply:
fixed